packages/marshaller/src/dashy.ts

```diff
@@ -309,6 +354,62 @@ export class Dashy extends SplitPanel {
             label: "Clear",
             execute: () => {
                 this.clear();
+                this.pushHistory();
+            }
+        });
+
+        commands.addCommand("dash_bookmark", {
+            label: "Bookmark State",
+            execute: () => {
+                const that = this;
+                if (typeof window["require"] !== "undefined") {
+                    window["require"](["../../packages/layout/lib-umd/index", "../../packages/form/lib-umd/index"], (layout, form) => {
```

Comment:
The only package allowed to do dynamic loading is "composite".

Reply:
FYI (and this is not 100% supported in rollup.js yet) the correct way to do this in the future is:
```
    import("@hpcc-js/layout").then({Modal} =>{
        ....
    });
```